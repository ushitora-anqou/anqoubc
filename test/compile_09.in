((1.1 + 2.5) * 0.5 - 1.0) / 2.0;
