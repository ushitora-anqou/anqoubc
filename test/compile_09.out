0.400000f
